espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Server.java

```diff
@@ -37,7 +37,7 @@
 public class Server extends NanoHTTPD {
 
     private Router router;
-    private static final int DEFAULT_PORT = 8080;
+    private static final int DEFAULT_PORT = 6791;
```

Comment:
It looks `systemPort` doesn't actually get used in this class. It's always set to `DEFAULT_PORT`.